In [1]:
import numpy as np
import pandas as pd
import scipy.stats as st

In [2]:
# Sample given data 
data = np.array([1.13, 1.55, 1.43, 0.92, 1.25, 
                 1.36, 1.32, 0.85, 1.07, 1.48, 
                 1.20, 1.33, 1.18, 1.22, 1.29])

n = len(data)                 # sample size
mean = np.mean(data)          # sample mean
s = np.std(data, ddof=1)      # sample standard deviation


In [4]:
alpha = 0.01
df = n - 1
SE = s / np.sqrt(n)

t_crit = st.t.ppf(1 - alpha/2, df)

ci_t = (mean - t_crit*SE, mean + t_crit*SE)

print(ci_t)


(np.float64(1.0901973384384906), np.float64(1.3871359948948425))


In [6]:
sigma = 0.2   # given population std
SE_known = sigma / np.sqrt(n)

z_crit = st.norm.ppf(1 - alpha/2)

ci_z = (mean - z_crit*SE_known, mean + z_crit*SE_known)

print(ci_z)


(np.float64(1.1056514133957607), np.float64(1.3716819199375725))


Conclusion

1.Both confidence intervals are very close, meaning the estimate of the mean durability is stable and reliable.

2.The t-distribution interval is slightly wider than the z-interval.
  a.This happens because with the sample standard deviation, there is extra uncertainty (population std is not known).
  b.T-distribution accounts for this by “stretching” the interval a little.

3.The manufacturer can be 99% confident that the true mean durability of the print-heads lies between ~1.09 and 1.39 million characters.